.github/workflows/auto-release-docker.yml

```diff
@@ -0,0 +1,32 @@
+name: Upload Docker Image to GHCR
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-image:
+    name: Release NeuroConv Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.MY_GITHUB_USERNAME }}
+          password: ${{ secrets.MY_GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          push: true  # Push is a shorthand for --output=type=registry
+          tags:
+            - ghcr.io/CatalystNeuro/NeuroConv:latest
+            - ghcr.io/CatalystNeuro/NeuroConv:0.3.1  # TODO, figure out how to auto-set this
```

Comment:
@CodyCBakerPhD something like this could work I think
```
      - name: Extract version
        id: extract-version
        run: |
          version=$(sed -n 's/version = "\(.*\)"/\1/p' ./src/neuroconv/version.py)
          echo "packageversion=${version}" >> "$GITHUB_ENV"
      - name: Build and push
        uses: docker/build-push-action@v3
        with:
          push: true
          tags:
            - ghcr.io/CatalystNeuro/NeuroConv:latest
            - ghcr.io/CatalystNeuro/NeuroConv:${{ env.packageversion }}
```

Reply:
of course, we would have to create a `version.py` file

Reply:
Neat idea - could probably also leverage a one-liner Python `-c` call like

```
version=$( python -c "from nwbinspector import get_package_version; print(get_package_version(package_name="neuroconv"))" )
echo "packageversion=${version}" >> "$GITHUB_ENV"
```

Reply:
This was resolved in a way reliant on the GitHub tags for each release